backend/app/assets/stylesheets/spree/backend/sections/_style_guide.scss

```diff
@@ -106,6 +106,15 @@
       border: 1px solid darken($color-7, 10%);
     }
   }
+
+  table {
+    th, td {
+      &.actions {
+        border-right: 1px solid #cee1f4 !important;
+        border-bottom: 1px solid #cee1f4 !important;
```

Comment:
I'm ok with this as the problem is due to the original - wrong - implementation.